README.md

```diff
@@ -1 +1,31 @@
-# swayswap-demo
\ No newline at end of file
+# ⚡️ SwaySwap
+[![License](https://img.shields.io/github/license/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+[![Issues Open](https://img.shields.io/github/issues/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+[![Github Forks](https://img.shields.io/github/forks/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+[![Github Stars](https://img.shields.io/github/stars/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+
+Short description
+    
+![SwaySwap Interface](cover.png)
+
+## About the Project
```

Comment:
Can remove this for now?

Reply:
I commented it out. Do you want me to fully remove it? @luizstacio